src/nvim/log.h

```diff
@@ -67,6 +67,30 @@
        __VA_ARGS__)
 #  endif
 
+# if __linux__
+#  include <execinfo.h>
+#  define LOG_CALLSTACK(prefix) \
```

Comment:
Why? This is for the developer to use for temporary debugging. What good will it do if LOG_CALLSTACK compiles but doesn't do anything?